workflows/python/sdk/README.md

```diff
@@ -0,0 +1,103 @@
+- verify_inventory_activity: This activity checks the state store to ensure that there is enough inventory present for purchase.
+- update_inventory_activity: This activity removes the requested items from the state store and updates the store with the new remaining inventory value.
+- requst_approval_activity: This activity seeks approval from Manager, if payment is greater than 50000 USD.
+
+### Run the order processor workflow
+
+1. Open a new terminal window and navigate to `order-processor` directory: 
+
+<!-- STEP
+name: Install requirements
+-->
+
+```sh
+cd ./order-processor
+pip3 install -r requirements.txt
+```
+
+<!-- END_STEP -->
+2. Run the console app with Dapr: 
+<!-- STEP
+name: Running this example
+expected_stdout_lines:
+  - "There are now 89 cars left in stock"
+  - "Purchase of item is  Completed"
+output_match_mode: substring
+background: true
+timeout_seconds: 30
+sleep: 15
+-->
+
+```sh
+cd ./order-processor
+dapr run --app-id order-processor --components-path ../../../components/ -- python3 app.py
+```
+
+<!-- END_STEP -->
+
+3. Expected output
+
+```
+==========Begin the purchase of item:==========
+To restock items, type 'restock'.
+To exit workflow console app, type 'exit'.
+Enter the name of one of the following items to order: paperclip, cars, computers: cars
+How many cars would you like to purchase? 11
+Starting order workflow, purchasing 11 of cars
+INFO:NotifyActivity:Received order b903d749cd814e099f06ebf4a56a2f90 for 11 cars at $165000 !
+INFO:VerifyInventoryActivity:Verifying inventory for order b903d749cd814e099f06ebf4a56a2f90 of 11 cars
+INFO:VerifyInventoryActivity:There are 100 Cars available for purchase
+INFO:RequestApprovalActivity:Requesting approval for payment of 165000 USD for 11 cars
+(ID = b903d749cd814e099f06ebf4a56a2f90) requires approval. Approve? [Y/N] y
+INFO:NotifyActivity:Payment for order b903d749cd814e099f06ebf4a56a2f90 has been approved!
+INFO:ProcessPaymentActivity:Processing payment: b903d749cd814e099f06ebf4a56a2f90 for 11 cars at 165000 USD
+INFO:ProcessPaymentActivity:Payment for request ID b903d749cd814e099f06ebf4a56a2f90 processed successfully
+INFO:UpdateInventoryActivity:Checking inventory for order b903d749cd814e099f06ebf4a56a2f90 for 11 cars
+INFO:UpdateInventoryActivity:There are now 89 cars left in stock
+INFO:NotifyActivity:Order b903d749cd814e099f06ebf4a56a2f90 has completed!
+Workflow completed! Result: Completed
+  Purchase of item is  Completed
+```
+
+### View workflow output with Zipkin
+
+For a more detailed view of the workflow activities (duration, progress etc.), try using Zipkin.
+
+1. Launch Zipkin container - The [openzipkin/zipkin](https://hub.docker.com/r/openzipkin/zipkin/) docker container is launched on running `dapr init`.
+
+2. View Traces in Zipkin UI - In your browser go to http://localhost:9411 to view the workflow trace spans in the Zipkin web UI. The order-processor workflow should be viewable with the following output in the Zipkin web UI. 
```

Comment:
```suggestion
1. View Traces in Zipkin UI - In your browser go to http://localhost:9411 to view the workflow trace spans in the Zipkin web UI. The order-processor workflow should be viewable with the following output in the Zipkin web UI. Note: the [openzipkin/zipkin](https://hub.docker.com/r/openzipkin/zipkin/) docker container is launched on running `dapr init`.
```
I still don't think we need a step because it is not actionable for the majority of users.  Let's make the note in step 2, and make that the main step.  ok?

Reply:
Changed as you suggested.